components/asset/Header.tsx

```diff
@@ -178,46 +185,72 @@ const AssetHeaderInfo: FC<Props> = ({ symbol }) => {
   );
 
   return (
-    <Grid
-      sx={{ bgcolor: 'components.bg' }}
-      width="100%"
-      p="24px"
-      boxShadow={({ palette }) => (palette.mode === 'light' ? '0px 4px 12px rgba(175, 177, 182, 0.2)' : '')}
-    >
-      <Grid item container mb="24px" alignItems="center">
-        <DropdownMenu
-          label={t('Asset')}
-          options={options}
-          onChange={onChangeAssetDropdown}
-          renderValue={<AssetOption assetSymbol={symbol} optionSize={22} selectedSize={30} />}
-          renderOption={(o: string) => <AssetOption option assetSymbol={o} optionSize={22} selectedSize={30} />}
-        />
-        {marketAccount && priceFeedAddress && (
-          <ExplorerMenu
-            symbol={symbol}
-            assetAddress={marketAccount.asset}
-            eMarketAddress={marketAccount.market}
-            rateModelAddress={marketAccount.interestRateModel.id}
-            priceFeedAddress={priceFeedAddress}
-            exaToken={marketAccount.symbol}
-          />
-        )}
-        <Typography sx={{ width: '100%' }} variant="dashboardMainSubtitle">
-          {assetDescription(symbol)}
-        </Typography>
-      </Grid>
-      <Grid item container spacing={4}>
-        {itemsInfo.map(({ label, value, underLabel, tooltipTitle }) => (
-          <ItemInfo
-            key={label.trim()}
-            label={label}
-            value={value}
-            underLabel={underLabel}
-            tooltipTitle={tooltipTitle}
+    <>
+      <Grid
+        sx={{ bgcolor: 'components.bg' }}
+        width="100%"
+        p="24px"
+        boxShadow={({ palette }) => (palette.mode === 'light' ? '0px 4px 12px rgba(175, 177, 182, 0.2)' : '')}
+      >
+        <Grid item container mb="24px" alignItems="center">
+          <DropdownMenu
+            label={t('Asset')}
+            options={options}
+            onChange={onChangeAssetDropdown}
+            renderValue={<AssetOption assetSymbol={symbol} optionSize={22} selectedSize={30} />}
+            renderOption={(o: string) => <AssetOption option assetSymbol={o} optionSize={22} selectedSize={30} />}
           />
-        ))}
+          {marketAccount && priceFeedAddress && (
+            <ExplorerMenu
+              symbol={symbol}
+              assetAddress={marketAccount.asset}
+              eMarketAddress={marketAccount.market}
+              rateModelAddress={marketAccount.interestRateModel.id}
+              priceFeedAddress={priceFeedAddress}
+              exaToken={marketAccount.symbol}
```

Comment:
I know this came from before, but what does this prop `exaToken` mean? are we passing always the same value, if that's the case it should be hardcoded, if not we should use a different name

Reply:
nevermind, I got confused. To avoid confusion we could use `exaVoucher
`